From 591b4ddaa2f7ab9b832904f4fcbd78e39fe9d355 Mon Sep 17 00:00:00 2001
From: "guoping.li" <guoping.li@amlogic.com>
Date: Thu, 2 Feb 2023 17:04:14 +0800
Subject: [PATCH 1/5] Amlbian: support Chinese simple language [1/1]

PD#SWPL-109095

Problem:
chinese charameter display abnormal

Solution:
apt-get install language-pack-zh-hans
apt-get install language-pack-gnome-zh-hans
apt install $(check-language-support) -y

support CHS will cause rootfs increase about 1.5G.
currently, default close this feature in config file.

change default as open.
dont submit, but include in auto build for QA

Verify:
display chinese folder name OK

Signed-off-by: guoping.li <guoping.li@amlogic.com>
Change-Id: I046c9ad21945b756ef237a6cc488f4c47465790a
---
 config/config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/config b/config/config
index fa7166d..d0a9c8a 100755
--- a/config/config
+++ b/config/config
@@ -121,7 +121,7 @@ SUPPORT_LANGUAGE_VALUES="ENG | CHS" # English | Chinese simple
 [[ -z $INSTALL_TYPE_RAW ]] && INSTALL_TYPE_RAW=no
 [[ -z $NO_APT_CACHER ]] && NO_APT_CACHER=yes
 [[ -z $SKIP_HEADER_DEB ]] && SKIP_HEADER_DEB=yes
-[[ -z $SUPPORT_LANGUAGE ]] && SUPPORT_LANGUAGE=ENG # TODO : should change to ENG before release
+[[ -z $SUPPORT_LANGUAGE ]] && SUPPORT_LANGUAGE=CHS # TODO : should change to ENG before release
 
 
 GITHUB_URL="https://github.com"
-- 
2.25.1

